rest/api_test.go

```diff
@@ -5558,7 +5558,7 @@ func TestPutTombstoneWithoutCreateAsDeletedFlagCasFailure(t *testing.T) {
 	// var UpdateXattrCallback func()
 	// ==========================
 
-	// Insert the below into UpdateXattr in bucket_gocb.go before the 'if isDelete && !supportsTombstoneCreation' check
+	// Insert the below into UpdateTombstoneXattr in bucket_gocb.go before the 'if isDelete && !supportsTombstoneCreation' check
```

Comment:
The location/naming of this changed in the PR. Given it's a recent feature, probably worth keeping this up to date for ease of manual testing.

```suggestion
	// Insert the below into UpdateTombstoneXattr in bucket_gocb.go before the 'if requiresBodyRemoval' check
```